Description: some tests fail on mips64el
 That architecture support improves over time - node 20.x branch has better support
 Meanwhile, let those tests fail.
Forwarded: not-needed
--- a/test/parallel/parallel.status
+++ b/test/parallel/parallel.status
@@ -65,6 +65,17 @@
 # https://github.com/nodejs/node/issues/54801
 test-debugger-heap-profiler: PASS, FLAKY
 
+[$arch==mips64el]
+test-worker-heapsnapshot-options: PASS,FLAKY
+test-cli-node-options: SKIP
+test-crypto-sign-verify: PASS,FLAKY
+test-freeze-intrinsics: PASS,FLAKY
+test-tls-root-certificates: PASS,FLAKY
+test-vm-global-non-writable-properties: PASS,FLAKY
+test-vm-global-setter: PASS,FLAKY
+test-vm-module-synthetic: PASS,FLAKY
+test-vm-strict-assign: PASS,FLAKY
+
 [$system==solaris] # Also applies to SmartOS
 # https://github.com/nodejs/node/issues/56726
 test-buffer-tostring-range: SKIP
